### ISSUE TITLE
Add Pear AI file and Folder context menu options: "open with Pear"

### ISSUE THREAD

**Issue by Krarilotus** [trypear/pearai-app#374]
Such a small thing, but making it more visible and convenient to use, adding the same context menu options as for other IDEs makes sense.
Right now opening a project is requiring the following steps:
- open the editor
- click on file or folder open
- manually navigating to a folder in your folder structure, which you have likely open and copy the path from another folder explorer window

Let's make this easier!
So my request is, that you add two check boxes to the installer, which enable you to add the IDE to the file menu options and the folder menu options to open a file or a folder with PearAI (Just "open with Pear" would do it...), as exists in the VS Code installation already.

**Comment by timknappe** [comment]
Good call, we should add this with #284, implementation turns out to be a bit more difficult since its not included in the installer in our case
